@xen-orchestra/defined/package.json

```
@@ -23,6 +23,7 @@
     "node": ">=6"
   },
   "scripts": {
-    "postversion": "npm publish"
+    "postversion": "npm publish",
+    "test": "node--test"
```

`node--test` is defined in the `test` package which is not listed as a dependency.

As the test in this package are using the native `node:test` module, I suspect you wanted to use `node --test` instead.

```suggestion
    "test": "node --test"
```

---

"node": ">=6" doesn't allow node:test?